src/components/sbb-form-field/sbb-form-field.scss

```diff
@@ -190,7 +190,17 @@
     }
   }
 
-  :host([floating-label]:not([floating-label='false'], :focus-within)[data-input-empty]) & {
+  // If floating-label is activated and there is no focus in it (except for select) and no popup is open
+  // and input is empty
+  // then apply the label transition.
+  :host(
+      [floating-label]:not(
+          [floating-label='false'],
+          :focus-within:not([data-input-type='sbb-select']),
```

Comment:
Why the exception on the select?
I expected that the `data-has-popup-open` would suffice :|